pkgs/games/anki/bin.nix

```diff
@@ -46,9 +47,10 @@ in
 
 if stdenv.isLinux then buildFHSUserEnv (appimageTools.defaultFhsEnvArgs // {
   name = "anki";
+  targetPkgs = pkgs: (with pkgs; [ xorg.libxkbfile krb5 ]);
```

Comment:
Why are these needed? Add a comment please.

Reply:
The new anki build refused to start without them, complaining about missing libraries. krb5 was needed for gssapi.

Reply:
Great, just add the explanations to the code